Softeq.XToolkit.WhiteLabel/Validation/Rules/LengthRule.cs

```diff
@@ -0,0 +1,38 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+
+namespace Softeq.XToolkit.WhiteLabel.Validation.Rules
+{
+    public class LengthRule : IValidationRule<string>
```

Comment:
Maybe it would be better to name it "StringLengthRule" or something like that? (to mark, that this will only work with Strings)